src/main/java/com/unorganized/dao/ITaskListDAO.java

```diff
@@ -0,0 +1,10 @@
+package com.unorganized.dao;
+
+import com.unorganized.dto.TaskDTO;
+import java.util.List;
+
+public interface ITaskListDAO {
+    List<TaskDTO> getTasks();
+    void addTask(TaskDTO task);
+    void deleteTask(String taskName);
+}
```

Comment:
Use interfaces to define contract and layers. This will allow polymorphism - multiple DAOs can be created (for example, once database is implemented, or a mock DAO for testing), and they can just be switched around whenever needed.